source/help/getting-started/organizing-conversations.rst

```diff
@@ -8,17 +8,19 @@ Channels are used to organize conversations across different topics. There are t
 
 Public Channels
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Public Channels are open to everyone on a team. New team members are automatically a part of two Public Channels: Town Square and Off-Topic.  
+Public Channels are open to everyone on a team. New team members are automatically added to two Public Channels when they sign up: Town Square and Off-Topic.
 
 Private Groups
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Private Groups are for sensitive topics and are only visible to select team members. Any member of a Private Group can add additional members. Group members can choose to leave at any time, but only the group owner or `Team Admin <http://docs.mattermost.com/help/getting-started/managing-members.html#user-roles>`_ can remove other members. 
+Private Groups are for sensitive topics and are only visible to select team members. Any member of a Private Group can add additional members. Group members can choose to leave at any time, but only the group owner or `Team Admin <http://docs.mattermost.com/help/getting-started/managing-members.html#user-roles>`_ can remove other members.
 
-Direct Messages
+Direct Messages and Group Messages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Direct Messages are for conversations between two users. These conversations are only visible to the people involved. They are special because they are not team specific - you can message any user on the site by going to the Direct Messages "More" menu, and choosing "All Users" in the dropdown menu there. If this option is not available, it has been turned off by your System Administrator.
+Direct Messages are for conversations between two people. Group Messages are Direct Messages that have conversations among three or more people. Both are visible only to the people involved.
 
-When you open a Direct Message channel with someone who is not on your current team, they will show up in the bottom section of the Direct Messages list under the "Outside this team" indicator. When you switch between teams, the list will update to reflect whether or not users belong to the current team. 
+Use a Direct Message when you want a private conversation with one other person. Use a Group Message when you want to quickly set up a conversation with up to seven other participants. If you want more than that, you must create a Private Group.
+
+If your System Administrator has allowed it, you can start a Direct Message or a Group Message with anyone from another team. When you open a Direct Message with someone who is not on your current team, they show up in the bottom section of the Direct Messages list under the "Outside this team" indicator. When you switch between teams, the list is updated to reflect whether or not users belong to the current team.
```

Comment:
This part changed, we removed the "Outside this team" line so can probably remove this section